platform/pom.xml

```diff
@@ -178,7 +178,12 @@
             <dependency>
                 <groupId>net.minidev</groupId>
                 <artifactId>json-smart</artifactId>
-                <version>1.1.1</version>
+                <version>${json-smart.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>net.minidev</groupId>
+                <artifactId>asm</artifactId>
+                <version>1.0.2</version>
```

Comment:
People should be moving to Boon as opposed to using json-smart. I added a property for json-smart because we have transitive dependencies using it and I wanted to keep all of those versions in sync.